src/main/scala/leon/purescala/ScalaPrinter.scala

```diff
@@ -39,6 +41,13 @@ class ScalaPrinter(opts: PrinterOptions,
       case m @ FiniteMap(els, k, v)  => p"Map[$k,$v]($els)"
 
       case InfiniteIntegerLiteral(v) => p"BigInt($v)"
+      case StringLiteral(v) =>
```

Comment:
It is exactly the same. I think I had code compilation issues at some point so I wrote this code here as well.
I removed it.